public/locales/en/dial.json

```diff
@@ -9,8 +9,10 @@
     "hazardous": "Hazardous (301+)",
     "invalid": {
         "lastTime": "This sensor is not up to date. The last time it reported an accurate AQI was ",
-        "at": " at ",
+        "atPlural": " at ",
+        "atSingular": " at ",
         "learnMore": ". To learn more about why a sensor might not be reporting, check out our ",
-        "aboutPage": "About Page."
+        "aboutPage": "About Page.",
+        "neverReported": "This sensor has never reported an accurate AQI"
```

Comment:
Do we need a period here?

Reply:
We don't because the beginning of the next string starts w/ a period (to account for the fact that there's no period in the time-string).